jinahub/indexers/storage/MongoDBStorage/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.3'
```

Comment:
Why is this part of the Executor main dir? It should be part of tests, right?

Reply:
isn't `docker-compose` also in the executor main dir of postgresstorage?

Reply:
it shouldn't be there